### PR TITLE
Support includeAll Query

### PR DESCRIPTION
**Pull request by dplewis**
https://github.com/parse-community/parse-server/pull/4838

provides shorthand for include('*')

### CODE REVIEW

**Comment by flovilmart** [comment]
Can we use the '*' as an include argument? 
```
query.include('*')
```
and perhaps includeAll being a shorthand for that? This will be easier to handle on the server also instead of a new parameter.

**Comment by dplewis** [comment]
@flovilmart This is what you mean right?

**Comment by flovilmart** [comment]
I mean to treat include all as just using `.include('*')` which is a change that isn't required in the SDK in the end if we support it server side :)

**Comment by dplewis** [comment]
I'll make an update server side. While preserving https://github.com/parse-community/parse-server/pull/4838

File: src/ParseQuery.js [hunk 1]
```diff
@@ -1313,6 +1315,10 @@ class ParseQuery {
   /**
    * Includes nested Parse.Objects for the provided key.  You can use dot
    * notation to specify which fields in the included object are also fetched.
+   * 
+   * If you want to include all nested Parse.Objects pass in '*'
+   * <pre>query.include('*');</pre>
```
Comment (flovilmart) [inline src/ParseQuery.js]: perhaps we need to put a version number here?

Comment (dplewis) [reply src/ParseQuery.js]: Which version? 2.1.0?

Comment (flovilmart) [reply src/ParseQuery.js]: I mean server version?

Comment (dplewis) [reply src/ParseQuery.js]: I have no idea mate, I can try to backtrack

Comment (flovilmart) [reply src/ParseQuery.js]: Ok ok, so basically it already works if you pass include All in the rest SDK.

Comment (dplewis) [reply src/ParseQuery.js]: Yes that is correct. Basically this won't work until the next release unless I re-add includeAll to this SDK

Comment (flovilmart) [reply src/ParseQuery.js]: Yeah, let’s just use the * notation then we’ll perhaps be able to expand on with reg ex etc...

Comment (flovilmart) [reply src/ParseQuery.js]: ❤️

**Comment by dplewis** [comment]
@flovilmart can this be merged?

File: src/ParseQuery.js (hunk 1 above)
Comment (GoGross) [reply src/ParseQuery.js]: How best can we update the guides simultaneously with the API docs

**Pull request merged**